osu.Game.Tests/Visual/Online/TestSceneWikiMarkdownContainer.cs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using Markdig.Syntax.Inlines;
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics.Containers.Markdown;
+using osu.Game.Online.API;
+using osu.Game.Overlays;
+using osu.Game.Overlays.Wiki.Markdown;
+
+namespace osu.Game.Tests.Visual.Online
+{
+    public class TestSceneWikiMarkdownContainer : OsuTestScene
+    {
+        private TestMarkdownContainer markdownContainer;
+
+        [Cached]
+        private readonly OverlayColourProvider overlayColour = new OverlayColourProvider(OverlayColourScheme.Orange);
+
+        [Cached]
+        private readonly IAPIProvider api = new DummyAPIAccess();
```

Comment:
is there a reason for doing this locally? generally the dummy api is already available in the base test scene implementation